frontend/src/queries/QueryEditor/InlineHogQLEditor.tsx

```diff
@@ -38,6 +38,11 @@ export function InlineHogQLEditor({ value, onChange }: InlineHogQLEditorProps):
             >
                 {value ? 'Update HogQL expression' : 'Add HogQL expression'}
             </LemonButton>
+            <div className="text-right">
+                <a href="https://github.com/PostHog/meta/issues/86" target={'_blank'}>
```

Comment:
Shouldn't this use `<Link />`?

Reply:
Maybe. Probably. We definitely need to use `<Link />` for all internal URLs, though in the case of `target='_blank'` urls, it doesn't really matter, and I've seen many around the codebase. We should probably change it though.

I quickly merged the PR earlier to give the beta users who might see the interface a bit of help with HogQL. I'll be doing many more changes to this textarea over the sprint, so I'll fix the link then. Thanks!